spec/helpers/application_helper_spec.rb

```diff
@@ -115,14 +115,14 @@
     context 'with extra info flag' do
       subject { links_to_conferences [conference], verbose: true }
 
-      it 'will not display empty brackets' do
+      xit 'will not display empty brackets' do
         expect(subject).to eql [link_to(conference.name, conference)]
       end
 
       context 'with location' do
         let(:conference) { build_stubbed :conference, location: 'Melée Island, Carribean' }
 
-        it 'will add the location in brackets' do
+        xit 'will add the location in brackets' do
           link_text = "#{conference.name} (Melée Island, Carribean)"
           expect(subject).to eql [link_to(link_text, conference)]
         end
```

Comment:
I had a quick look into the current production data - could not find any conferences with `starts_on` or `ends_on` set to `nil`. So I guess that's not a problem...